src/Raven.Server/Documents/TransactionOperationsMerger.cs

```diff
@@ -826,7 +827,7 @@ private enum PendingOperations
                 var modifiedSize = llt.NumberOfModifiedPages * Constants.Storage.PageSize;
 
                 var canCloseCurrentTx = previousOperation == null || previousOperation.IsCompleted;
-                if (canCloseCurrentTx)
+                if (canCloseCurrentTx || llt.Environment.Options.ForceUsing32BitsPager || PlatformDetails.Is32Bits)
```

Comment:
Can we extract this computation to the constructor? 
I want to make it easy to ignore

Reply:
Also, what about the behavior a bit below, where we do the check again in:

```
// if I can't close the tx, this means that the previous async operation is still in progress
// and there are incoming requests coming in. We'll accept them, to a certain limit
if (modifiedSize < _maxTxSizeInBytes)
    continue; // we can still process requests at this time, so let's do that...
```

Reply:
I'm not sure what should be the behavior in the merger, I don't want to start rejecting requests maybe it is better if @aviviadi  or @haludi will look into that?
_maxTxSizeInBytes controls the max transaction size and I think it should be limited to 2MB for 32 bit the same way smuggler is limiting its batches.

Reply:
actually the merger is limiting this to 4MB for 32bit so we can remove this entirly